### ISSUE TITLE
color profile Properties can get out of sync with PhET-iO

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/gas-properties#145]
Now that we have Tandem.global and it shows up in Studio (courtesy of https://github.com/phetsims/tandem/issues/94), I see that there a conflict between ColorProfile's `profileNameProperty` (a string) and GasPropertiesGlobalOptions `projectorModeEnabledProperty` (a boolean).  Both Properties are `phetioReadOnly: false`. 

If the client sets `profileNameProperty` to `'projector'`, then `projectorModeEnabledProperty` does not change to `true`.

**Comment by pixelzoom** [comment]
This seems like something that should be handled by joist.ProjectorModeCheckBox, not every client.   So transferring this issue from gas-properties to joist, and marking as blocking publication.
